android.md

```
@@ -0,0 +1,392 @@
+* RCT2 assets;
+* OpenRCT2 data files;
+* Installed OpenRCT2 app.
+
+As of writing, the minimum required Android version is Android 7.0 ([SDK 24](https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/openrct2-android/app/build.gradle#L21)).
+
+### RCT2 assets
+The Android version (just like other versions) requires assets from the original RCT2 game to work. This means you need to copy the RCT2 installation folder to your phone.
+
+There a few ways to get these files:
+* a) Install RCT2 on a Windows computer and navigate to the installation folder.
+* b) If you own RCT on either Steam or GOG, but do not have access to a Windows computer: there's a few ways to get the files, like on MacOS described [here](https://docs.openrct2.io/en/latest/installing/installing-on-macos.html#from-a-gog-com-offline-installer) and the steps for Linux are described [here](https://docs.openrct2.io/en/latest/installing/installing-on-linux-bsd.html#getting-the-rct2-files-required).
```

markdownlint-cli2 (0.20.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)